core/domain/config_domain.py

```diff
@@ -104,11 +104,6 @@ class ConfigProperty(object):
     - whitelisted_email_senders.
     """
 
-    def refresh_default_value(self, default_value):
```

Comment:
It's a bit shocking that this has been here for so long :P Thanks for removing it!

Reply:
Welcome :)